src/cp.js

```diff
@@ -153,6 +171,42 @@ function cpdirSyncRecursive(sourceDir, destDir, currentDepth, opts) {
   } // for files
 } // cpdirSyncRecursive
 
+function major(rdev) {
```

Comment:
An explanatory comment might be good, since this is pretty niche and not very obvious